cmd/watch.go

```diff
@@ -88,6 +88,9 @@ func watch() {
 	ticker := time.NewTicker(5 * time.Second)
 	defer ticker.Stop()
 
+	fmt.Printf("Timeout is %d", timeout)
+	return
```

Comment:
This return should be removed

Reply:
yes, it was part of my local debug code. missed it. thank you